src/templates/src/index.ts.squirrelly

```diff
@@ -1,51 +1,66 @@
 import {
-{{ @if (it.aptitudes.includes("clipboard")) }}
+{{ @if (it.aptitudes.includes('clipboard')) }}
   clipboard,
 {{ /if }}
-{{ @if (it.aptitudes.includes("filesystem")) }}
+{{ @if (it.aptitudes.includes('filesystem')) }}
   filesystem,
 {{ /if }}
-{{ @if (it.aptitudes.includes("keyboard")) }}
+{{ @if (it.aptitudes.includes('keyboard')) }}
   keyboard,
 {{ /if }}
-{{ @if (it.aptitudes.includes("network")) }}
+{{ @if (it.aptitudes.includes('network') || it.aptitudes.includes('filesystem')) }}
```

Comment:
@dmorrison-olive There turned out to be one broken case with this new logic so you were right 🙃